config/resources/domain.json

```diff
@@ -143,26 +117,9 @@
           "cardinality": "many"
         }
       },
-      "features": [
-        "include-uri"
-      ],
+      "features": ["include-uri"],
       "new-resource-base": "http://lblod.data.gift/concepts/"
     },
-    "concepts": {
```

Comment:
I see here more stuffs were removed, not that it's bad but I wonder if we did also cleanup the frontend as well? as it gets a bit more confusing with this major refactor, stuffs are kept / renamed and we leave bugs along the way (example: https://github.com/lblod/app-mow-registry/pull/90) , better to focus one one thing per PR imho